lib/improver/cli/apply_emos_coefficients.py

```diff
@@ -130,58 +130,104 @@ def main(argv=None):
 
     args = parser.parse_args(args=argv)
 
+    # Load Cubes
     current_forecast = load_cube(args.forecast_filepath)
     coeffs = load_cube(args.coefficients_filepath)
 
-    original_current_forecast = current_forecast.copy()
+    # Process Cube
+    result = process(current_forecast, coeffs, args)
```

Comment:
shouldn't this be something like:
```suggestion
    result = process(current_forecast, coeffs, **vars(args))
```